engine/src/main/java/org/terasology/engine/module/ModuleManager.java

```diff
@@ -125,6 +134,18 @@ public ModuleManager(Config config, List<Class<?>> classesOnClasspathsToAddToEng
         this(config.getNetwork().getMasterServer(), classesOnClasspathsToAddToEngine);
     }
 
+    private ModuleMetadataJsonAdapter newMetadataReader() {
+        final ModuleMetadataJsonAdapter metadataJsonAdapter;
+        metadataJsonAdapter = new ModuleMetadataJsonAdapter();
```

Comment:
Why not do this in one line?
```suggestion
        final ModuleMetadataJsonAdapter metadataJsonAdapter = new ModuleMetadataJsonAdapter();
```